### PR TITLE
Feature/project setup

### LINKED ISSUE #11: Create project structure
- [x] Add main structure of django project (without any specific apps). Project root should contain a specific folder for all apps. Configuration folder (that stores urls, settings, and middleware configs) should be named "config";
- [x] All environment variables (e.g. DB host and port )should be moved to a separate .env file, which will not be pushed in the repository (for security reasons);
- [x] Setup postgres as project DB;
- [x] Add docker-compose file to run environment locally;
- [x] Add requirements file.

### PR DESCRIPTION
**Pull request by borisoid**
Fixes #11

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# DataBase
+/data/
```
Comment (Zomba4okk) [inline .gitignore:2]: The concrete path is `./data/`

File: config/settings.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
```
Comment (Zomba4okk) [inline config/settings.py]: DRF library is not connected

File: config/settings.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+# https://docs.djangoproject.com/en/3.1/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': env('db_name'),
+        'USER': env('db_user'),
+        'PASSWORD': env('db_password'),
+        'HOST': env('db_server_host'),
+        'PORT': env('db_server_port'),
```
Comment (Zomba4okk) [inline config/settings.py]: Variable names are usually capitalized.
E.g. `db_name` - not right, `DB_NAME` - right

**Review by Zomba4okk — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (borisoid) [reply .gitignore:2]: Nope, my path appears to be correct.<br>
Screenshot_1
Screenshot_3

**Review by Zomba4okk — APPROVED** [review]

**Pull request merged**